ansible/roles/celery/tasks/main.yml

```diff
@@ -20,22 +21,27 @@
   sudo: yes
   tags: celery
 
+- name: Pull latest covalic metrics Docker image
+  sudo: yes
+  command: docker pull girder/covalic-metrics:latest
```

Comment:
I read that earlier today but it wasn't clear to me from those docs how to pull an image down. All their examples seem to be about running containers. Any ideas?